builtin/mainmenu/dlg_create_world.lua

```diff
@@ -15,6 +15,12 @@
 --with this program; if not, write to the Free Software Foundation, Inc.,
 --51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
+-- load default settings
+core.setting_set("mg_flags","trees, caves")
+core.setting_set("mgv5_spflags","blobs")
+core.setting_set("mgv6_spflags","biomeblend, jungles, mudflow")
+core.setting_set("mgv7_spflags","mountains, ridges")
```

Comment:
We could load the default from minetest.conf. At beginning these are the defaults from minetest.conf.example. When you change the flags in the dialog you could store them in minetest.conf. Then the players have always their own default settings.